code/modules/mechanics/MechanicMadness.dm

```diff
@@ -347,6 +347,7 @@
 	plane = PLANE_NOSHADOW_BELOW
 	w_class = W_CLASS_TINY
 	level = 2
+	burn_possible = FALSE
```

Comment:
```suggestion
```
let them burn 😈

Reply:
It makes molitz b burns impossible, I designed modern molitz b for people to use mechcomp teleporters/pipes to move it in and out (see the comments on molitz b stating that the intended way is to teleport the molitz b inside and outside of the chamber)

Reply:
Also forgot to mention but its very on the table to make the nonflammability only apply to wrenched in teleport comps + pipe comps as those are really the only ones relevant here

Reply:
For me, learning how to handle molitz b was at a time when mechcomp teleporters were not able to survive burn chamber heats. It made for a much more frustrating experience than it would otherwise have been. Other engineers get discouraged fast after I tell them that the mechcomp solution they came up with, after I told them about the molitz blinking requirements, with doesn't work. So imo that's a big plus for this change.

Maybe there are balance concerns with mechanics being able to set up teletraps to burn chambers, but they can already teleport people to crushers, with some of them existing off station, so is it really that big a concern?